src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/RunReportPageTests.kt

```diff
@@ -139,7 +139,7 @@ class RunReportPageTests : IntegrationTest()
     }
 
     @Test
-    fun `only report runners can get running report logs`()
+    fun `report runner can get running report logs`()
```

Comment:
Opportunistic test rename - this test doesn't live up the 'only' bit - that non-runners can's get report logs is tested elsewhere.